src/content/translations/id/developers/docs/transactions/index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Transaksi
+description: Gambaran umum tentang transaksi Ethereum – cara kerjanya, struktur datanya, dan cara mengirimnya melalui aplikasi.
+lang: id
+sidebar: true
+---
+
+Transaksi adalah instruksi yang ditandatangani secara kriptografik dari akun. Akun akan menginisiasi transaksi untuk memperbarui state jaringan Ethereum. Bentuk transaksi paling sederhana adalah mentransfer ETH dari satu akun ke akun yang lain.
+
+## Prasyarat {#prerequisites}
+
+Untukmembantu Anda memahami halaman ini dengan lebih baik, kami menyarankan Anda membaca terlebih dahulu [Akun](/developers/docs/accounts/) dan [pengantar Ethereum](/developers/docs/intro-to-ethereum/) kami.
+
+## Apa itu transaksi? {#whats-a-transaction}
+
+Transaksi Ethereum mengacu pada aksi yang dimulai oleh akun dengan kepemilikan eksternal, dengan kata lain, akun yang dikelola oleh manusia, bukan kontrak. Sebagai contoh, jika Bob mengirimkan 1 ETH ke Alice, akun Bob harus didebit dan akun Alice harus dikredit. Aksi yang mengubah state ini terjadi dalam sebuah transaksi.
+
+![Diagram yang menunjukkan sebuah transaksi menyebabkan perubahan state](../../../../../developers/docs/transactions/tx.png) _Diagram diadaptasi dari [Ethereum EVM yang diilustrasikan](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
```

Comment:
This image is broken in deploy preview but it is correct - (same path as [Chinese docs](https://ethereum.org/zh/developers/docs/transactions/))